sql/2021/privacy/most_common_cmps_for_iab_tcf_v2.sql

```diff
@@ -0,0 +1,34 @@
+#standardSQL
+# Counts of CMPs using IAB Transparency & Consent Framework
+# cf. https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#tcdata
+
+WITH totals AS (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(0) AS total_websites
+  FROM
+    `httparchive.pages.2021_07_01_*`
+  GROUP BY
+    _TABLE_SUFFIX
+)
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  JSON_VALUE(JSON_VALUE(payload, "$._privacy"), '$.iab_tcf_v2.data.cmpId') AS cmpId,
+  COUNT(0) AS nb_websites,
+  total_websites,
+  COUNT(0) / ANY_VALUE(total_websites) AS pct_websites
+FROM
+  `httparchive.pages.2021_07_01_*`
+JOIN
+  totals
+USING (_TABLE_SUFFIX)
+WHERE
+  JSON_VALUE(JSON_VALUE(payload, "$._privacy"), '$.iab_tcf_v2.data.cmpId') IS NOT NULL
+GROUP BY
+  client,
+  total_websites,
+  cmpId
+ORDER BY
+  client ASC,
```

Comment:
As `ASC` is default there's no need to explicitly say this and we generally tend not to bother with it. Though appreciate some might prefer to be explicit.

Reply:
I don't see a value changing it to implicit here.

Reply:
Main value would be in being consistent across chapters, so there's not an expectation of lack of explicit ASC means. It's not one we've enforced before to be honest, but looking across the current queries I can only find 12 instances of being explicit with ASC so definitely in the minority.